tests/quick_test.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import unittest
+
+from google.appengine.ext import testbed
+from webtest import TestApp
+
+sys.path.append('./main')
+
+
+class AppTest(unittest.TestCase):
+  def setUp(self):
+    # Wrap the app with WebTest's TestApp.
+    self.testbed = testbed.Testbed()
+    self.testbed.activate()
+    self.testbed.init_memcache_stub()
+    self.testbed.init_datastore_v3_stub()
+    from main import app
+    self.testapp = TestApp(app)
+
+  def tearDown(self):
```

Comment:
```suggestion
  def tear_down(self):
```

Reply:
No -- this has to override the parent method of this name.